package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "version": "5.5.2",
+    "version": "5.6.0",
```

Comment:
There's no need to change the version number in such a PR. One a tag is made the CI will commit the changed version number back to the repository automatically.